### PR TITLE
Add instructions for installing Toolkit from package

### LINKED ISSUE #541: [Content Bug] toolkit installation page does not mention available packages
toolkit installation page does not mention available packages

## What do the docs say now?
Nothing

## What should the docs say?
Packages are available for Debian, Ubuntu, RedHat

# Page affected

https://docs.timescale.com/timescaledb/latest/how-to-guides/hyperfunctions/install-toolkit/#install-timescaledb-toolkit

# Version affected
1.3.0
[Which version of the site you noticed the bug on]

# Subject matter expert (SME)
@svenklemm

### PR DESCRIPTION
**Pull request by charislam**
# Description

TimescaleDB toolkit is now available as an Ubuntu, Debian, or RPM package. This PR adds package installation instructions to the Toolkit install guide.

Also includes:
- Minor fixes to align with branding and style guide
- Spell out reference links so Vale can check for broken links

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #541

### CODE REVIEW

**Comment by charislam** [comment]
@svenklemm I started revising the Toolkit installation page to mention the Debian/Ubuntu/RPM packages. Disclaimer: I hacked together the procedures from minimal understanding of Linux so they may be utterly wild. Corrections much appreciated!

**Comment by svenklemm** [comment]
Looks good, maybe put in link how to install timescaledb repo for folks that don't have it yet

File: timescaledb/how-to-guides/hyperfunctions/install-toolkit.md [hunk 1]
```diff
@@ -17,12 +18,60 @@ ALTER EXTENSION timescaledb_toolkit UPDATE;
...
-If you are hosting your own TimescaleDB database you install the Toolkit
-extension from the command prompt.
+If you're hosting your own TimescaleDB database, you can install Toolkit as an
+RPM, Debian, or Ubuntu package. You can also build Toolkit from source.
 
 <procedure>
 
-### Installing Toolkit on self-hosted TimescaleDB
+### Installing Toolkit on Red Hat-based systems
+These instructions use the `dnf` package manager on RHEL, CentOS, and Fedora.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/hyperfunctions/install-toolkit.md:27]: Move this line out of the procedure.

File: timescaledb/how-to-guides/hyperfunctions/install-toolkit.md [hunk 2]
```diff
@@ -17,12 +18,60 @@ ALTER EXTENSION timescaledb_toolkit UPDATE;
...
+
+1.  Make sure you have installed TimescaleDB and created a TimescaleDB
+    repository in your `yum` repos. For more information, see [the installation
```
Comment (Loquacity) [inline timescaledb/how-to-guides/hyperfunctions/install-toolkit.md:30]:
```suggestion
    repository in your `yum` repo.d directory. For more information, see [the installation
```

??

File: timescaledb/how-to-guides/hyperfunctions/install-toolkit.md [hunk 3]
```diff
@@ -17,12 +18,60 @@ ALTER EXTENSION timescaledb_toolkit UPDATE;
...
+    ```sql
+    CREATE EXTENSION timescaledb_toolkit;
+    ```
+
+</procedure>
+
+<procedure>
+
+### Installing Toolkit on Debian-based systems
+These instructions use the `apt` package manager on Debian and Ubuntu.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/hyperfunctions/install-toolkit.md:51]: Move out of the procedure.

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: timescaledb/how-to-guides/hyperfunctions/install-toolkit.md (hunk 2 above)
Comment (charislam) [reply timescaledb/how-to-guides/hyperfunctions/install-toolkit.md:30]:
```suggestion
    repository in your `yum` `repo.d` directory. For more information, see [the installation
```

**Comment by charislam** [comment]
> Looks good, maybe put in link how to install timescaledb repo for folks that don't have it yet

Thanks @svenklemm! The repo install instructions are at the same link as the install TimescaleDB instructions; I reworded the link text so hopefully that's a bit clearer now

**Review by Loquacity — APPROVED** [review]

**Pull request merged**